Classes/ObjectExplorers/FLEXObjectExplorerViewController.m

```diff
@@ -36,25 +50,30 @@ @interface FLEXMethodBox : NSObject
 @implementation FLEXMethodBox
 @end
 
-static const NSInteger kFLEXObjectExplorerScopeNoInheritanceIndex = 0;
-static const NSInteger kFLEXObjectExplorerScopeIncludeInheritanceIndex = 1;
-
 @interface FLEXObjectExplorerViewController () <UISearchBarDelegate>
```

Comment:
I think we could get the structure of the state on this class to a better place but it's not urgent. I'd also like to kill the inheritance used to customize the object explorers and do it via configuration. Would be cool to expose public hooks on the framework so you could customize display for different classes.